snap/snapcraft.yaml

```diff
@@ -328,6 +328,28 @@ parts:
       - -usr/share/doc
       - -usr/share/man
 
+  bom:
+    after:
```

Comment:
Does it really need to be built after all components?

Reply:
Yes, because we need to pull in the revision from which we have built the binaries. See line 75 for an example

Reply:
I see, thanks.